src/data.js

```diff
@@ -332,6 +332,34 @@ module.exports = [
       'Gatsby',
     ],
   },
+  {
+    name: 'Colin Principe',
+    description:
+      'Software Guild graduate, Full stack developer, Java backend, Angular/React frontend.',
+    url: 'http://principe.io/uses',
```

Comment:
This isn't a uses page

Reply:
URL changed, was having issues with my hosting.  Latest commit resolves properly.
[https://github.com/FatherOfCurses/awesome-uses/commit/9d2d15026f52119d3335fbfcca759cad6945ca75](Latest commit)